redis/jobs/consumers/consumer_scan_mod_on_virus_total.go

```diff
@@ -92,9 +89,5 @@ func ScanModOnVirusTotalConsumer(ctx context.Context, payload []byte) error {
 		go integrations.NewVersion(util.ReWrapCtx(ctx), version)
 	}
 
-	go storage.DeleteModArch(ctx, mod.ID, mod.Name, version.ID, "Combined")
-	go storage.DeleteModArch(ctx, mod.ID, mod.Name, version.Version, "Combined")
-	go postgres.Delete(ctx, modArch)
```

Comment:
Are these no longer necessary?

Reply:
`consumers.UpdateModDataFromStorage` and `consumers.UpdateDBFromModVersionJSONFileConsumer` require the original uploaded file to be kept, since they are running `validation.ExtractModInfo`, which is why the reference to that file is also kept in the `Version.Key` field, even though it is not exposed through the API anymore.